onnxruntime/core/optimizer/graph_transformer_utils.cc

```diff
@@ -233,7 +238,7 @@ InlinedVector<std::unique_ptr<GraphTransformer>> GenerateTransformers(
 
       transformers.emplace_back(std::make_unique<ConvActivationFusion>(cpu_cuda_rocm_acl_armnn_eps));
 
-      transformers.emplace_back(std::make_unique<GeluFusion>(cpu_cuda_rocm_eps));
+      transformers.emplace_back(std::make_unique<GeluFusion>(cpu_cuda_rocm_dml_eps));
```

Comment:
some of our L2/L3 optimizers contains EP specific code, like if (is_cuda)...,  please check whether it matches DML ep's expectation. we might want to refactor that part.